README.md

```diff
@@ -21,5 +21,46 @@ client build <directory> --output my-workspace
 ### Template content in a directory and push to a registry location
 `client build <directory> --push --destination localhost:5000/myartifacts:latest`
 
+## Getting Started
+
+1. Create a new directory. 
+2. Add the content to be uploaded in the directory (can be files of any content types).
```

Comment:
Do we want to be more explicit on the files that the user should be creating?

Reply:
Can we address this in the use case?

Reply:
sure and we can always loop back as necessary to update